pkgs/os-specific/bsd/netbsd/pkgs/makeMinimal.nix

```
@@ -22,7 +23,9 @@ mkDerivation {
 
   postPatch = ''
     patchShebangs $COMPONENT_PATH/configure
-    ${make.postPatch}
+
+    # make needs this to pick up our sys make files
+    export NIX_CFLAGS_COMPILE+=" -D_PATH_DEFSYSPATH=\"$out/share/mk\""
```

Can we use `appendToVar` here as well, for consistency?

```suggestion
    appendToVar NIX_CFLAGS_COMPILE -D_PATH_DEFSYSPATH="$out/share/mk"
```